lib/monitor/watch.js

```diff
@@ -195,6 +195,51 @@ function ignoredFileTypesForFind(dir) {
   return ' \\( ' + paths.join(' -or ') + ' \\) -or ';
 }
 
+// Inspired by http://underscorejs.org/docs/underscore.html
+function throttle(func, wait, options) {
+  var context, args, result;
+  var timeout = null;
+  var previous = 0;
+  if (!options) options = {};
+  var later = function() {
+    previous = options.leading === false ? 0 : new Date().getTime();
+    timeout = null;
+    result = func.apply(context, args);
+    if (!timeout) context = args = null;
+  };
+  return function() {
+    var now = new Date().getTime();
+    if (!previous && options.leading === false) previous = new Date().getTime();
+    var remaining = wait - (now - previous);
+    context = this;
+    args = arguments;
+    if (remaining <= 0 || remaining > wait) {
```

Comment:
This line always allows the first request through - which we didn't actually want for the delay param.